src/Paynow/Exception/PaynowException.php

```diff
@@ -23,7 +23,7 @@ public function __construct($message = '', $code = 0, $body = null)
     }
 
     /**
-     * @return array|null
+     * @return Error[]
```

Comment:
from the flow I presume that return type should nullable cause of two reasons:
*. when `$body` is `null` the constructor skips L15
* `$errors` property is not initialized within a value, even with an empty array

Reply:
@emilleszczak2 can you add `|null` on the docblock `@return` tag?

Reply:
Sure